Update text-field.md (#265)
FlutterFlow/flutterflow-documentation: docs/resources/control-flow/user-interactivity/forms/form-widgets/text-field.md

Here's an example of displaying the TextField value in a Text widget in realtime:

1. Select the TextField widget, move to the **Properties Panel** **> Additional Properties >** enable the **Update Page On Change** property.

> 2. Also, set the **Update Delay (ms)**, which determines the time after the user has stopped typing for the page to refresh its UI. For example, if the *Update Delay (ms)* value is set to 2000 ms (2 seconds), the page will update 2 seconds after the user is done typing. For this example, let's set it to 0 ms.

Reviewer: ```suggestion
2. Also, set the **Update Delay (ms)**, which specifies the time interval after the user stops typing before the page refreshes its UI. For example, if the *Update Delay (ms)* value is set to 2000 ms (2 seconds), the page will update 2 seconds after the user stops typing. For this example, let's set it to 0 ms.
```
